infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/MachineResolver.java

```diff
@@ -78,8 +81,9 @@ private void normalizeMemory(Container container, InternalMachineConfig machineC
     if (ramLimit == 0) {
       machineConfig.getAttributes().put(MEMORY_LIMIT_ATTRIBUTE, defaultSidecarMemoryLimitBytes);
     }
+    // Use plugin_publisher/plugin_name to find overriding of memory limit.
     String overriddenSidecarMemLimit =
-        wsAttributes.get(format(SIDECAR_MEMORY_LIMIT_ATTR_TEMPLATE, pluginId));
+        wsAttributes.get(format(SIDECAR_MEMORY_LIMIT_ATTR_TEMPLATE, pluginPublisherName));
```

Comment:
Please make sure that java doc of this constant[1] is up to date and all usages are correct, I suppose here should be `pluginPublisherName/pluginName` instead of pluginPublisherName

[1] https://github.com/garagatyi/che/blob/publisher2/wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java#L122

Reply:
Fixed docs, checked usage - used correctly everywhere